test_integration/2_orchestrator_config_override_suite_test.go

```diff
@@ -0,0 +1,77 @@
+package test_integration
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/suite"
+	"strings"
+	"testing"
+)
+
+type OrchestratorConfigOverrideSuite struct {
+	BaseDockerComposeTestSuite
+}
+
+func NewOrchestratorConfigOverrideSuite() *OrchestratorConfigOverrideSuite {
+	s := &OrchestratorConfigOverrideSuite{}
+	s.GlobalRunIdentifier = globalTestExecutionId
+	s.SuiteRunIdentifier = strings.ToLower(strings.Split(uuid.New().String(), "-")[0])
+	return s
+}
+
+func (s *OrchestratorConfigOverrideSuite) SetupSuite() {
+	rawDockerComposeFilePath := "./common_assets/docker_compose_files/orchestrator-node-with-custom-start-command.yml"
+	s.Context, s.Cancel = context.WithCancel(context.Background())
+
+	orchestratorConfigFile1 := s.commonAssets("nodes_configs/2_config.yaml")
+	orchestratorConfigFile2 := s.commonAssets("nodes_configs/2_config_override.yaml")
+	orchestratorStartCommand := fmt.Sprintf("bacalhau serve --config=%s --config=%s", orchestratorConfigFile1, orchestratorConfigFile2)
+	extraRenderingData := map[string]interface{}{
+		"OrchestratorStartCommand": orchestratorStartCommand,
+	}
+	s.BaseDockerComposeTestSuite.SetupSuite(rawDockerComposeFilePath, extraRenderingData)
+
+	_, err := s.executeCommandInDefaultJumpbox(
+		[]string{
+			"mkdir",
+			"-p",
+			fmt.Sprintf("/app/%s", s.SuiteRunIdentifier),
+		})
+	s.Require().NoErrorf(err, "Error creating a suite tmp folder in the jumpbox node: %q", err)
```

Comment:
the same logic needs to be repeated in all test suites. Does it make sense to introduce a SuiteBuilder type or something similar with sane defaults and allows overriding certain behaviour when needed?

Reply:
Good catch. The copy paste laziness caught up with me last week.
Code is fixed, and I actually removed some of it that is not needed any more.